src/Rating.soy

```diff
@@ -3,10 +3,46 @@
 /**
  * This renders the main element.
  * @param? elementClasses
+ * @param? label
  * @param id
- * @skipUpdates elementClasses
  */
 {template .render}
 	<div id="{$id}" class="rating component{$elementClasses ? ' ' + $elementClasses : ''}">
+		{if $label}
```

Comment:
On the most recent version of Metal.js we're using [incremental dom](https://github.com/google/incremental-dom/), so dividing the template into sub templates to improve updates is not necessary anymore. You can still do it for organization purposes for big templates though, I'm just letting you know about it :)